etna/records/views/choosers.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import url
 from django.core.paginator import Page
 from django.shortcuts import Http404
+from django.urls import re_path as url
```

Comment:
Since the url() function is deprecated and removed in 4.0, could we use _re_path_ instead of _as url_ as alias here does not look necessary.

Reply:
Yes, Doing it this way will save changing the code elsewhere, the code uses URLs everywhere.

Reply:
one single change is better than changing many files.

Reply:
Yes, as discussed yesterday, we don't have URL's everywhere and it a change in this file itself.